.github/maintainers_guide.md

```diff
@@ -97,7 +105,7 @@ Releasing can feel intimidating at first, but rest assured: if you make a mistak
 
     - **External**
 
-      - **Bot Developer Hangout** (`dev4slack.slack.com`) in **#slack-api**
+      - **Slack Community Hangout** (`community.slack.com/`) in **#lang-javascript**. Include a link to the package on NPM as well as the release notes.
```

Comment:
this is a smaller discussion point - should we keep the NPM uppercase here, use "npm", or `npm`? 🤔 or none of the above?

Reply:
Or maybe we can just use the URL, i.e. "Include a link to the package on npmjs.com/package/@slack/<package>"